backend/recovery_orchestrator_enhanced.py

```
@@ -0,0 +1,1042 @@
+                "recurrence_count": event.recurrence_count,
+                "confidence": event.confidence,
+                "step": step_idx,
+            }
+
+            # Simple condition evaluation
+            for var, value in context.items():
+                condition = condition.replace(var, str(value))
+
+            # Evaluate (safe eval for simple conditions)
+            # In production, use a proper expression parser
+            result = eval(condition, {"__builtins__": {}}, {})
```

Ruff (0.14.10)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)